Formula/swift-format.rb

```diff
@@ -2,18 +2,18 @@ class SwiftFormat < Formula
   desc "Formatting technology for Swift source code"
   homepage "https://github.com/apple/swift-format"
   url "https://github.com/apple/swift-format.git",
-    tag:      "0.50200.1",
-    revision: "f22aade8a6ee061b4a7041601ededd8ad7bc2122"
+    tag:      "0.50300.0",
+    revision: "12089179aa1668a2478b2b2111d98fa37f3531e3"
   license "Apache-2.0"
   version_scheme 1
   head "https://github.com/apple/swift-format.git", branch: "main"
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "d8f72c33efc125e2904e1bec2c8942cca75d75cf81dcab7fcf08ba124af16170" => :catalina
   end
 
-  depends_on xcode: ["11.4", :build]
+  depends_on xcode: ["12.0", :build]
+  depends_on macos: :big_sur
```

Comment:
It doesn't depend on Big Sur though, this is a bug and it needs to be fixed.

Reply:
I think the issue is on the homebrew 11.5 build machine.  Ive reverted a few changes so that test can be run again, but Apple doesn't claim anything is required to build this other than Xcode 12.0+, so there shouldn't really be a difference between macOS 10.15 and 11.0.  My home machine has been updated to 11.0 so I'm not able to verify that it works as expected on 10.15.  From https://github.com/apple/swift-format:

> The SwiftSyntax version in use must match the toolchain version, so you should check out and build swift-format from the branch that is compatible with the version of Swift you are using.

Can we verify that Homebrew's 11.5 machine has `xcode-select -p` pointing to the Xcode 12.0 application and not the system installed Command Line Tools?  Is there a way for formulaes to explicitly require a specific Swift toolchain?

Reply:
I found other formulaes that had this block of code and that seems to have resolved the issues on 10.15!

```
  # The bottles are built on systems with the CLT installed, and do not work
  # out of the box on Xcode-only systems due to an incorrect sysroot.
  pour_bottle? do
    reason "The bottle needs the Xcode CLT to be installed."
    satisfy { MacOS::CLT.installed? }
  end
```